utilities/conv_gefs2datm/conv_gefs2datm.fort.csh

```diff
@@ -0,0 +1,52 @@
+#! /bin/csh -x
+#-- hyun-chul.lee@noaa.gov
+
+module load NetCDF/4.5.0
+
+set hdir = /scratch2/NCEPDEV/climate/Hyun-Chul.Lee/GEFS
+set sdir = ${hdir}/Reanal
+set tdir = ${hdir}/GEFS2DATM
+set wdir = ${hdir}/Work
+set grb2 = /home/Hyun-Chul.Lee/Tools/wgrib2
```

Comment:
This points to a personal home directory with a binary. 

Please update it by loading the appropriate module:

`module load wgrib2`